<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>910</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>271</width>
      <height>781</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>741</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>460</y>
      <width>581</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1162</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_open_db"/>
    <addaction name="action_new_db"/>
    <addaction name="action_save_as"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_edit_preferences"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="action_new_dataset"/>
    <addaction name="action_new_object"/>
    <addaction name="separator"/>
    <addaction name="action_analyze_dataset"/>
    <addaction name="separator"/>
    <addaction name="action_import_dataset"/>
    <addaction name="action_export_dataset"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuData"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open_db"/>
   <addaction name="action_new_db"/>
   <addaction name="action_save_as"/>
   <addaction name="action_new_dataset"/>
   <addaction name="action_new_object"/>
   <addaction name="action_export_dataset"/>
   <addaction name="action_import_dataset"/>
   <addaction name="action_analyze_dataset"/>
   <addaction name="action_edit_preferences"/>
  </widget>
  <action name="action_open_db">
   <property name="icon">
    <iconset>
     <normaloff>icon/open.png</normaloff>icon/open.png</iconset>
   </property>
   <property name="text">
    <string>Open DB</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="icon">
    <iconset>
     <normaloff>icon/saveas.png</normaloff>icon/saveas.png</iconset>
   </property>
   <property name="text">
    <string>Save AS</string>
   </property>
  </action>
  <action name="action_new_db">
   <property name="icon">
    <iconset>
     <normaloff>icon/newdb.png</normaloff>icon/newdb.png</iconset>
   </property>
   <property name="text">
    <string>New DB</string>
   </property>
  </action>
  <action name="action_new_dataset">
   <property name="icon">
    <iconset>
     <normaloff>icon/newdataset.png</normaloff>icon/newdataset.png</iconset>
   </property>
   <property name="text">
    <string>New Dataset</string>
   </property>
  </action>
  <action name="action_new_object">
   <property name="icon">
    <iconset>
     <normaloff>icon/newobject.png</normaloff>icon/newobject.png</iconset>
   </property>
   <property name="text">
    <string>New Object</string>
   </property>
  </action>
  <action name="action_export_dataset">
   <property name="icon">
    <iconset>
     <normaloff>icon/export.png</normaloff>icon/export.png</iconset>
   </property>
   <property name="text">
    <string>Export Dataset</string>
   </property>
   <property name="toolTip">
    <string>Export Dataset</string>
   </property>
  </action>
  <action name="action_import_dataset">
   <property name="icon">
    <iconset>
     <normaloff>icon/import.png</normaloff>icon/import.png</iconset>
   </property>
   <property name="text">
    <string>Import Dataset</string>
   </property>
   <property name="toolTip">
    <string>Import Dataset</string>
   </property>
  </action>
  <action name="action_analyze_dataset">
   <property name="icon">
    <iconset>
     <normaloff>icon/analyze.png</normaloff>icon/analyze.png</iconset>
   </property>
   <property name="text">
    <string>Analyze Dataset</string>
   </property>
   <property name="toolTip">
    <string>Analyze Dataset</string>
   </property>
  </action>
  <action name="action_edit_preferences">
   <property name="icon">
    <iconset>
     <normaloff>icon/preferences.png</normaloff>icon/preferences.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="toolTip">
    <string>Preferences</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset>
     <normaloff>icon/about.png</normaloff>icon/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
